librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -808,6 +809,9 @@ typedef void (*on_background_error_cb)(void*, crocksdb_backgrounderrorreason_t,
                                        crocksdb_status_ptr_t*);
 typedef void (*on_stall_conditions_changed_cb)(void*, const crocksdb_writestallinfo_t*);
 
+typedef void (*logv_cb)(
```

Comment:
move this closer to `crocksdb_logger_create_from_impl` declaration. rename it `crocksdb_logger_logv_cb`.

Reply:
Done

Reply:
Done